### DISCUSSION TITLE
Adding a delay to release dates

### DISCUSSION THREAD

**Discussion by Jauntiness** [itsToggle/plex_debrid#139] [Ideas]
Hi,

I experienced that adding newly released episodes can be tricky or buggy. 
E.g. for the series "See" adding Season 1-2 will work perfectly but trying to add newly released episodes by season03 will add episodes by series "See No Evil". If I try the same, e.g. 12 hours later it will work without any problem, since links for "See" can be found. Also, at a later point, it would be much more likely to find my wished different resolution versions.

My idea was to add an option for adding optional delays to release dates/times. So the tool will soonest, e.g. 12 hours after release date, start looking for links. 
That way everyone could have some impact on choosing between speed vs. quality for adding new episodes.

But maybe I am the only one experiencing this as a bug?

**Comment by itsToggle** [comment]
Hey, I'll add an option for the script to continue searching for desired qualities (#131).
I will also try to rework the scraping mechanism, so that no wrong shows will be downloaded.
I've created an issue from this and added the enhancement tag, a custom delay is a good idea non the less :)

**Reply by Jauntiness** [reply]
Cool, sounds like some great improvements will follow! Looking forward :)

**Reply by itsToggle** [reply]
hey, the issue of wrong shows being downloaded should now be fixed :)

**Reply by Jauntiness** [reply]
Nice! I will feedback you if I should find something else!
